actionview/lib/action_view/renderer/renderer.rb

```diff
@@ -19,10 +19,15 @@ def initialize(lookup_context)
 
     # Main render entry point shared by Action View and Action Controller.
     def render(context, options)
+      render_to_object(context, options).body
+    end
+
+    # Main render entry point shared by Action View and Action Controller.
+    def render_to_object(context, options)
```

Comment:
Should this be nodoc?

Reply:
I think so.  I wasn't sure exactly how public this should be.  I'll remove comments and put `:nodoc:` for now.  Thank you for reviewing! 🙇‍♂️